app/src/main/java/org/mozilla/fenix/tabstray/browser/TabGroupAdapter.kt

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray.browser
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.HORIZONTAL
+import mozilla.components.concept.tabstray.Tabs
+import mozilla.components.concept.tabstray.Tab as TabsTrayTab
```

Comment:
is this an inline alias?

Reply:
Yes, this is an import alias.  We did it here since there are two definition of `Tab`.

Reply:
Gotcha ok. I haven't seen them in imports before so this is pretty cool!